src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,41 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
```

Comment:
```suggestion
export const MovieCard = ({ movie: { imgUrl, title, description, imdbUrl } }) => (
```
just FYI - you may do nested destructuring assignment